app/scripts.babel/friends.js

```diff
@@ -145,6 +147,9 @@ init({ friends: true, cachedRanks: true }, ({ friends, cachedRanks }) => {
     const newRanklist = renderRanklist(header, friends);
     ranklist.parentElement.replaceChild(newRanklist, ranklist);
     ranklist = newRanklist;
+
+    Array.from(document.querySelectorAll(SELECTOR_TO_HIDE))
```

Comment:
This only needs to be done once. Instead of this, do something like `ranklist.parentElement.style.display = ''` outside of `replaceRanklist`.